packages/validate/lib/index.js

```diff
@@ -190,6 +168,25 @@ const updateEventArray = (events, updates) => {
   );
 };
 
+const normalizeVersion = (version) => {
+  if (/^[0-9]$/.test(version)) {
+    return `${version}.0.0`;
+  } else if (/^[0-9]+\.[0-9]$/.test(version)) {
+    return `${version}.0`;
+  } else if (/^[0-9]+\.[0-9]+\.[0-9]+$/.test(version)) {
```

Comment:
The first regex doesn't allow multiple digits as in the major version.
The second regex doesn't allow multiple digits in the minor version.

Reply:
rebased